### PR TITLE
Addition to bonfire

### PR DESCRIPTION
**Pull request by lbnesquik**

### CODE REVIEW

File: code/game/objects/structures/bonfire.dm [hunk 1]
```diff
@@ -116,7 +118,7 @@
 		var/obj/item/stack/S = F.split(1)
 		if(S)
 			S.forceMove(src)
-			to_chat(user, "<span class='warning'>You add \the [new_fuel] to \the [src].</span>")
+			to_chat(user, "<span class='warning'>You add one\the [new_fuel] to \the [src].</span>")
```
Comment (EvilJackCarver) [inline code/game/objects/structures/bonfire.dm]: "you add onethe fuel..."

File: code/game/objects/structures/bonfire.dm [hunk 2]
```diff
@@ -154,6 +156,13 @@
 		return FALSE
 	return TRUE
 
+/obj/structure/bonfire/examine(mob/user)
+	var/X = get_fuel_amount()
+	user << "The fire has [X] burning logs in it."
+	if(grill)
+		user << "The [src] has a crude grill plate over it."
+	if(grill)
+		user << "The [src] has a crude grill plate over it."
```
Comment (EvilJackCarver) [inline code/game/objects/structures/bonfire.dm]: Duplicate?

Comment (lbnesquik) [reply code/game/objects/structures/bonfire.dm]: Yes. Correct. Fixing.

File: code/game/objects/structures/bonfire.dm (hunk 1 above)
Comment (lbnesquik) [reply code/game/objects/structures/bonfire.dm]: There is a space behind it so it should be fine, besides, \the means it remove it, doesn't it?

Comment (EvilJackCarver) [reply code/game/objects/structures/bonfire.dm]: As far as I am aware, \the prints "the" for non proper items (those without a capital name)

**Pull request merged**